operations/before.go

```diff
@@ -194,6 +194,21 @@ func requireAtLeastOneBool(flags ...string) cli.BeforeFunc {
 	}
 }
 
+// If the bool flag is present check the required string flags are present as well
+func requireFlagsForBool(boolFlag string, requiredStringFlags ...string) cli.BeforeFunc {
```

Comment:
Could you make this generic for any type of flags? You can check c.IsSet for all of them

Reply:
This function was removed